sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -261,18 +264,183 @@ def _is_json_content_type(content_type: str) -> bool:
     return False
 
 
+def _build_sentinel(setting: Union[str, Tuple[str, str]]) -> Tuple[str, str]:
+    try:
+        key, label = setting
+    except IndexError:
+        key = setting
+        label = EMPTY_LABEL
+    if "*" in key or "*" in label:
+        raise ValueError("Wildcard key or label filters are not supported for refresh.")
+    return key, label
+
+
+def _is_retryable_error(error: HttpResponseError) -> bool:
+    """Determine whether the service error should be silently retried after a backoff period, or raised.
+    Don't know what errors this applies to yet, so just always raising for now.
+    """
+    return error.status_code in [429]  # Back off on throttling error
```

Comment:
Found these retriable errors in dotnet sdk. Would this be a good reference to specify retriable errors?
https://github.com/Azure/azure-sdk-for-net/blob/f81281597d2de54928522d5e07091b7165d3fd4a/sdk/core/Azure.Core/src/ResponseClassifier.cs#L20